src/server/common/tabs/secrets/views/partials/create-form.njk

```diff
@@ -1,10 +1,10 @@
 <h2 class="govuk-heading-l govuk-!-margin-bottom-2">Create secret</h2>
 
 <p>
-  Create a secret in {{ environment | title }}.
+  Create a secret in {{ environment | title }}
 </p>
 
-<form action="{{ routeLookup('services/{serviceId}/secrets/{environment}/create', {
+<form action="{{ routeLookup(serviceOrTestSuite + 's/{serviceId}/secrets/{environment}/create', {
```

Comment:
Is this meant to be putting  the `s` on the end of serviceOrTestSuite? Im guessing its pluralizing it?

Reply:
Correct. I'll see if I can use the pluralise function